lib/main_publiccloud.pm

```diff
@@ -164,6 +164,8 @@ sub load_latest_publiccloud_tests {
         loadtest "publiccloud/upload_image", run_args => $args;
     } elsif (check_var('PUBLIC_CLOUD_AHB', 1)) {
         loadtest 'publiccloud/ahb', run_args => $args;
+    } elsif (get_var('PUBLIC_CLOUD_NEW_INSTANCE_TYPE')) {
+        loadtest("publiccloud/bsc_1205002", run_args => $args);
```

Comment:
Sorry, but I see in master these lines for bsc_1205002 already present: 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/4aa0a3929e672f5eb4a43ae198e206c945a1b8e2/lib/main_publiccloud.pm#L46.

May be my visualization is buggy :thinking: ?